assets/sites/default/settings.php

```diff
@@ -154,23 +154,32 @@ function _data_starter_validates($variable = '') {
 // KEY for dkan health status.
 $conf['dkan_health_status_health_api_access_key'] = 'DKAN_HEALTH';
 
-// Add tracking codes for Google Analytics.
-if (isset($conf['gaClientTrackingCode']) && $conf['gaClientTrackingCode'] != 'UA-XXXXX-Y') {
-  $conf['googleanalytics_account'] = $conf['gaClientTrackingCode'];
-}
-elseif (isset($conf['gaNuCivicTrackingCode']) && $conf['gaNuCivicTrackingCode'] != 'UA-XXXXX-Z') {
-  $conf['googleanalytics_account'] = $conf['gaNuCivicTrackingCode'];
-}
-
-if (isset($conf['gaNuCivicTrackingCode']) &&
-  $conf['googleanalytics_account'] != $conf['gaNuCivicTrackingCode'] &&
-  $conf['gaNuCivicTrackingCode'] != 'UA-XXXXX-Z') {
-  $conf['googleanalytics_codesnippet_after'] = "ga('create', '" . $conf['gaNuCivicTrackingCode'] . "', 'auto', 'nucivicTracker');ga('nucivicTracker.send', 'pageview');";
-}
-
 // Never disallow cli access via shield config.
 $conf['shield_allow_cli'] = 1;
 
+// Only enable GA codes in production. This was not put in the switch statement
+// below as it is production only setting.
+if (ENVIRONMENT == 'production') {
```

Comment:
@rhabbachi we already have a switch case for production a little bit below can we use that instead of duplicating here.

Also, will unsetting with else statement take care of value set in DB?